values.yaml

```diff
@@ -1,33 +1,43 @@
-octopus:
-  statefulSet:
-    annotations: {}
-    labels: {}
-  pods:
-    annotations: {}
-    labels: {}
+statefulSetLabel:
+  environment: prod
+
+podLabels:
+  environment: prod
+
   # The Octopus server image. Visit https://hub.docker.com/r/octopusdeploy/octopusdeploy for the available versions.
   image: octopusdeploy/octopusdeploy:2020.3.1-linux
```

Comment:
I just learned that we now tag the most recent release with `latest`, so that would be a good choice here.

Reply:
Sweet! Just changed it.